packages/common/src/components/NbConda.tsx

```diff
@@ -363,11 +363,11 @@ export class NbConda extends React.Component<ICondaEnvProps, ICondaEnvState> {
           environments: await this.props.model.environments
         };
         if (this.state.currentEnvironment === undefined) {
-          newState.environments.forEach(env => {
-            if (env.is_default) {
-              newState.currentEnvironment = env.name;
-            }
-          });
+          const defaultEnvironment =
+            newState.environments.find(env => env.is_default) ||
+            // in case no environment is_default just load the first environment
+            newState.environments[0];
```

Comment:
Is there always at least one environment available?

Reply:
No there can be zero environments or zero environments visible to the user.

Reply:
Thanks @costrouc.

So that means we have three possible situations:

1. environments is not empty and has some environment that is_default
   - In this case, no change in the user experience (before vs after this PR)
2. environments is not empty and has no environment that is_default
   - In this case, the UI loads the first environment in the list
3. environments is empty (which implies it has no environment that is_default)
   - In this case, no change in the user experience

The 3rd case isn't great, because it means the package panel is shown the user but it will be empty and none of the inputs will actually work. My first take at https://github.com/Quansight/conda-store/issues/240, I tried hiding the package panel when there was no default environment to load, but I ran into some styling issues. It seems like those are worth fixing in order to provide a more robust and error-free UI.

Closing this PR for now.